script.js

```diff
@@ -12,6 +12,116 @@ const headerListElement = document.getElementById('header_list');
 const buttonElement = document.getElementById('button_back');
 let menuIsActive = false;
 let submenuIsActive = false;
+const buttonRight = document.getElementById('button_right');
+const buttonLeft = document.getElementById('button_left');
+const carouselText = document.getElementById('carousel_text');
+const carouselBox = document.getElementById('carousel_box');
+const carouselElement = document.getElementById('carousel');
+
+let carouselTextObject = document.createDocumentFragment();
+carouselTextObject = carouselText;
+
+let images = Array.from(carouselBox.children);
+let slider = [];
+images.forEach(function(item, i, images) {
+    slider[i] = images[i].src; 
```

Comment:
Why not just use slider.push(item.src) why are you passing in some i thing whatever that is and then images.

Reply:
Changed